packages/core/admin/admin/src/translations/en.json

```diff
@@ -753,7 +753,7 @@
   "content-manager.form.Input.filters": "Enable filters",
   "content-manager.form.Input.hint.character.unit": "{maxValue, plural, one { character} other { characters}}",
   "content-manager.form.Input.hint.minMaxDivider": " / ",
-  "content-manager.form.Input.hint.text": "{min, select, undefined {} other {min. {min}}}{divider}{max, select, undefined {} other {max. {max}}}{unit}{br}{description}",
+  "content-manager.form.Input.hint.text": "{min, select, undefined {} other {min. (min: {min})}}{divider}{max, select, undefined {} other {max. (max: {max})}}{unit}{br}{description}",
```

Comment:
Thanks! The other translations looks good.

I think this was better before though as I find "min. (min: 5) / max. (max: 100) characters" (e.g.) a bit strange to read

Reply:
Thanks @jhoward1994,
This translation has now been reverted to previous state.

Reply:
Thanks @jhoward1994,
This translation has now been reverted to previous state.